data/sql/updates/pending_db_world/remove-black-pearl-ring-in-red-sack.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM item_loot_template WHERE Entry = 17969 AND Item = 6332;
```

Comment:
Always add backticks :)

```suggestion
DELETE FROM `item_loot_template` WHERE `Entry` = 17969 AND `Item` = 6332;
```